### PR TITLE
Fix star mask

### PR DESCRIPTION
**Pull request by tbaranes**
I went ahead and looked in this issue (#95). Apparently, it was just missing the right center.
Does that looks ok for you @lexrus ? It's working well in playground.

I think we can release the 1.2 with this fixed! :tada:

### CODE REVIEW

File: IBAnimatable/MaskDesignable.swift [hunk 1]
```diff
@@ -133,14 +133,12 @@ public extension MaskDesignable where Self: UIView {
...
     let path = UIBezierPath()
-    let radius = min(layer.bounds.size.width, layer.bounds.size.height) / 2 - borderWidth
+    let radius = min(bounds.size.width, bounds.size.height) / 2 - borderWidth
     let starExtrusion = radius / 2
-    
     let angleIncrement = CGFloat(M_PI * 2.0 / Double(points))
-    var angle:CGFloat = -CGFloat(M_PI / 2.0)
-    
+    let center = CGPointMake(bounds.width / 2.0, bounds.height / 2.0)
+    var angle: CGFloat = -CGFloat(M_PI / 2.0)
```
Comment (JakeLin) [inline IBAnimatable/MaskDesignable.swift]: Can be `var angle = -CGFloat(M_PI / 2.0)`

**Comment by JakeLin** [comment]
:+1: I have updated README.md and CHANGELOG.md ready for 1.2, once we get this one in, then we can release [1.2](https://github.com/JakeLin/IBAnimatable/milestones/1.2) 🎊

**Comment by tbaranes** [comment]
Ready to be merged, your call guys :+1:

**Comment by lexrus** [comment]
Hey @tbaranes 

I remember I have fixed this issue in another branch... I don't know.
Anyway, thank you.

But I have a question. What's the difference between the original `center` property of UIView and `CGPointMake(bounds.width / 2.0, bounds.height / 2.0)`.
BTW. I think we should use `CGPoint(x: CGFloat, y: CGFloat)` instead of `CGPointMake`.

**Comment by lexrus** [comment]
I found tons of `UIView.animateWithDuration(...` in my company projects. It's time to spread IBAnimatable. :dancers:

**Comment by tbaranes** [comment]
Good catch for the `CGPoint`, I miss it.
I didn't search further for the center since I didn't have enough time, the center of `UIView` was wrong. If anyone know why, I'm interested :)

**Comment by lexrus** [comment]
> The center is specified **within the coordinate system of its superview** and is measured in points. Setting this property changes the values of the frame properties accordingly.

Okay, I got it.

screen shot 2016-02-16 at 4 24 28 pm

**Comment by JakeLin** [comment]
@tbaranes thanks, well done.👍

@lexrus good to spread to the world. 👏

**Pull request merged**